packages-content-model/roosterjs-content-model-plugins/lib/tableEdit/editors/features/CellResizer.ts

```diff
@@ -0,0 +1,232 @@
+    editor: IStandaloneEditor,
+    td: HTMLTableCellElement,
+    table: HTMLTableElement,
+    isRTL: boolean,
+    isHorizontal: boolean,
+    onStart: () => void,
+    onEnd: () => false,
+    anchorContainer?: HTMLElement
+): TableEditFeature | null {
+    const document = td.ownerDocument;
+    const createElementData = {
+        tag: 'div',
+        style: `position: fixed; cursor: ${isHorizontal ? 'row' : 'col'}-resize; user-select: none`,
+    };
+    const zoomScale = editor.getZoomScale();
+
+    const div = createElement(createElementData, document) as HTMLDivElement;
+
+    (anchorContainer || document.body).appendChild(div);
+
+    const context: DragAndDropContext = { editor, td, table, isRTL, zoomScale, onStart };
+    const setPosition = isHorizontal ? setHorizontalPosition : setVerticalPosition;
+    setPosition(context, div);
+
+    const handler: DragAndDropHandler<DragAndDropContext, DragAndDropInitValue> = {
+        onDragStart,
+        // Horizontal modifies row height, vertical modifies column width
+        onDragging: isHorizontal ? onDraggingHorizontal : onDraggingVertical,
+        onDragEnd: onEnd,
+    };
+
+    const featureHandler = new DragAndDropHelper<DragAndDropContext, DragAndDropInitValue>(
+        div,
+        context,
+        setPosition,
+        handler,
+        zoomScale,
+        editor.getEnvironment().isMobileOrTablet
+    );
+
+    return { node: td, div, featureHandler };
+}
+
+interface DragAndDropContext {
+    editor: IStandaloneEditor;
+    td: HTMLTableCellElement;
+    table: HTMLTableElement;
+    isRTL: boolean;
+    zoomScale: number;
+    onStart: () => void;
+}
+
+interface DragAndDropInitValue {
+    cmTable: ContentModelTable | undefined;
+    anchorColumn: number | undefined;
+    anchorRow: number | undefined;
+    initialX: number;
+    allWidths: number[];
+}
+
+function onDragStart(context: DragAndDropContext, event: MouseEvent): DragAndDropInitValue {
+    const { td, onStart } = context;
+    const rect = normalizeRect(td.getBoundingClientRect());
+
+    // Get cell coordinates
+    const columnIndex = td.cellIndex;
+    const row = td.parentElement instanceof HTMLTableRowElement ? td.parentElement : undefined;
+    const rowIndex = row?.rowIndex;
+
+    if (rowIndex == undefined) {
+        return {
+            cmTable: undefined,
+            anchorColumn: undefined,
+            anchorRow: undefined,
+            initialX: 0,
+            allWidths: [],
+        }; // Just a fallback
+    }
+
+    const { editor, table } = context;
+
+    // Get current selection
+    const selection = editor.getDOMSelection();
+
+    // Select first cell of the table
+    editor.setDOMSelection({
+        type: 'table',
+        firstColumn: 0,
+        firstRow: 0,
+        lastColumn: 0,
+        lastRow: 0,
+        table: table,
+    });
+
+    // Get the table content model
+    const cmTable = getFirstSelectedTable(editor.createContentModel())[0];
```

Comment:
I have removed the API `createContentModel`, and replaced with a new one `getContentModelCopy`